src/metabase/models/collection.clj

```diff
@@ -324,40 +324,59 @@
             [:not-like :location (hx/literal (format "%%/%s/%%" (str visible-collection-id)))]))))))
 
 
-(s/defn effective-location-path :- (s/maybe LocationPath)
```

Comment:
For some of these that used `s/defn` rather than normal `defn` I just changed the existing definition to a private function and created a new function that calls the impl function